csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Testing;
```

Comment:
I'm curious.

This file, and several others, have strange name (the + and the ` character.)  Is that intentional? And why?

Reply:
@BillWagner This is what the default template has. The naming for this is made the same as the symbol metadata name.